src/models/queryTermGroupBy.js

```diff
@@ -0,0 +1,18 @@
+class QueryTermGroupBy {
+  /**
+   * [constructor]
+   * @param  {String} field
+   * @param  {Array}  terms
+   * @return {QueryTermGroupBy Class}
+   */
+    constructor(field, terms = []) {
+      this.field = field;
+      this.terms = terms;
+    }
+
+    toJsonAPI() {
+      return this.field;
+    }
+}
+
+export default QueryTermGroupBy;
```

Comment:
you also need to modify index.js to export `Query` and `QueryAggregates` models.